lib/broadway/message.ex

```diff
@@ -81,6 +81,27 @@ defmodule Broadway.Message do
     %Message{message | batch_mode: mode}
   end
 
+  @doc """
+  Configures the acknowledger of this message.
+
+  This function calls the `c:Broadway.Acknowledger.configure/3` callback to
+  change the configuration of the acknowledger for the given `message`.
+
+  This function can only be called if the acknowledger implements the `configure/3`
+  callback. If it doesn't, an error is raised.
+  """
+  @spec configure_ack(message :: Message.t(), options :: keyword) :: Message.t()
+  def configure_ack(%Message{} = message, options) when is_list(options) do
+    %{acknowledger: {module, ack_ref, ack_data}} = message
+
+    if function_exported?(module, :configure, 3) do
+      {:ok, ack_data} = module.configure(ack_ref, ack_data, options)
```

Comment:
Do we want to raise a better error if the acker doesn't return `{:ok, _}`?

Reply:
Nah.